packages/ethereum-contracts/contracts/utils/ForwarderBase.sol

```diff
@@ -25,12 +25,17 @@ abstract contract ForwarderBase {
     }
 
     function _forwardBatchCall(ISuperfluid.Operation[] memory ops) internal returns (bool) {
+        return _forwardBatchCall(ops, 0);
+    }
+
+    function _forwardBatchCall(ISuperfluid.Operation[] memory ops, uint256 valueToForward) internal returns (bool) {
```

Comment:
let's learn from the past and use overloaded names with caution, in this case `_forwardBatchCallWithValue` is readable at use-site without having to resort to reading the library code or any editor feature.